man/homectl.xml

```diff
@@ -0,0 +1,807 @@
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>homectl</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="req">COMMAND</arg>
+      <arg choice="opt" rep="repeat">NAME</arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>homectl</command> may be used to create, remove, change or inspect a user's home
+    directory. It's primarily a command interfacing with
+    <citerefentry><refentrytitle>systemd-homed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    which manages home directories of users.</para>
+
+    <para>Home directories managed by <filename>systemd-homed.service</filename> are self-contained, and thus
+    include the user's full metadata record in the home's data storage itself, making them easily migratable
+    between machines. In particular a home directory in itself describes a matching user record, and every
+    user record managed by <filename>systemd-homed.service</filename> also implies existance and
+    encapsulation of a home directory. The user account and home directory hence become the same concept. The
+    following backing storage mechanisms are supported:</para>
+
+    <itemizedlist>
+      <listitem><para>Individual LUKS2 encrypted loopback files for each user, located in
+      <filename>/home/*.home</filename>. At login the file systems contained in these files are mounted,
+      after the LUKS2 encrypted volume is attached. The user's password is identical to the encryption
+      passphrase of the LUKS2 volume. Access to data without preceeding user authentication is thus not
+      possible, not even for the systems administrator. This storage mechanism provides the strongest data
+      security and is thus recommended.</para></listitem>
+
+      <listitem><para>Similar, but the LUKS2 encrypted file system is located on regular block device, such
+      as an USB storage stick. In this mode home directories and all data they include are nicely migratable
+      between machines, simply by plugging the USB stick into different systems at different
+      times.</para></listitem>
+
+      <listitem><para>An encrypted directory using <literal>fscrypt</literal> on file systems that support it
+      (at the moment this is primarily <literal>ext4</literal>), located in
+      <filename>/home/*.homedir</filename>. This mechanism also provides encryption, but substantially
+      weaker than the two mechanisms described above, as most file system metadata is unprotected. Moreover
+      it currently does not support changing user passwords once the home directory has been
+      created.</para></listitem>
+
+      <listitem><para>A <literal>btrfs</literal> subvolume for each user, also located in
+      <filename>/home/*.homedir</filename>. This provides no encryption, but good quota
+      support.</para></listitem>
```

Comment:
... yeah, if the suffix is `.homedir` than "home directories" is the appropriate name. Conversely, if "homes" wins, than the suffix should be `.home`, yuck.

Reply:
Actually I used ".home" as suffix if home directories are files (i.e. are LUKS2 images) and ".homedir" if they are directories (i.e. are fscrypt, btrfs subvol or plain directory).

The ".homedir" is supposed to distuingish classic home dirs with no ~/.identity and those managed by homed that have it. During login /home/lennart.homedir is mounted to /home/lennart btw, to drop the suffix, similar how for the luks case /home/lennart.home is mounted to /home/lennart. This makes behaviour in these cases more alike: the dirs area available under their name during login only while the user is actually logged in.